Sources/tart/Commands/IP.swift

```diff
@@ -34,10 +34,20 @@ struct IP: AsyncParsableCommand {
     let vmConfig = try VMConfig.init(fromURL: vmDir.configURL)
     let vmMACAddress = MACAddress(fromString: vmConfig.macAddress.string)!
 
-    guard let ip = try await IP.resolveIP(vmMACAddress, resolutionStrategy: resolver, secondsToWait: wait) else {
+    let lock = try PIDLock(lockURL: vmDir.configURL)
+    if try lock.pid() == 0 {
```

Comment:
I think you can simplify this by using `vmDir.running()` instead of instantiating the `PIDLock` here.

Reply:
Oh! Forgot we extracted the logic in a method. 👌 Fixed in 2487a844f5b839d9d51d4564c092c87a5f865ea8